go-controller/Makefile

```diff
@@ -9,6 +9,8 @@ PKGS ?=
 GOPATH ?= $(shell go env GOPATH)
 TEST_REPORT_DIR?=$(CURDIR)/_artifacts
 export TEST_REPORT_DIR
+GO_DKR_IMG = golang:1.15.7-buster
+DOCKER_RUNNABLE ?= $(shell docker -v > /dev/null 2>&1 && ! printenv CI > /dev/null 2>&1; echo $$?)
```

Comment:
if we are going to skip CI please add a comment explaining it
The recommended ways is to check if $GITHUB_ACTIONS=true
> GITHUB_ACTIONS | Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.

https://docs.github.com/es/actions/reference/environment-variables

Reply:
@aojea To clarify, the we are not skipping running tests in CI. The tests will be run the regular way  as earlier but NOT inside of a container when in the CI environment. I am using the `CI` environment variable from https://docs.github.com/es/actions/reference/environment-variables to check if the tests are running inside CI environment and run the tests the earlier way and not inside of docker containers. I noticed that when the tests are run in a docker container when in the CI environment, some of the tests needing mount options are failing even though the docker container is run with the right linux capabilities. This would need to be investigated further but should not block the current PR from merging. 
@aojea per your suggestion, I will add a comment at Line 13 providing additional details